tests/functional/spec/features/repeating_sections/repeating_sections_with_hub_and_spoke.spec.js

```diff
@@ -155,27 +156,19 @@ describe("Feature: Repeating Sections with Hub and Spoke", () => {
 
       // Start section for first visitor
       $(HubPage.summaryRowLink("section")).click();
-      $(PrimaryPersonPage.submit()).click();
```

Comment:
Behaviour has changed, once a section is complete you will now need to use the section summary to navigate, you can't linearly step from one block to the next if that section was already completed